tortoise/backends/base/schema_generator.py

```diff
@@ -177,7 +177,9 @@ def _get_unique_constraint_sql(self, model: "Type[Model]", field_names: List[str
             fields=", ".join([self.quote(f) for f in field_names]),
         )
 
-    def _get_table_sql(self, model: "Type[Model]", safe: bool = True) -> dict:
+    def _get_table_sql(
```

Comment:
Why not get models_tables inside instead of pass in?

Reply:
> Why not get models_tables inside instead of pass in?

good point. I really missed this. models_tables refers to model._meta .db_table so this can be changed.